nexus/src/external_api/http_entrypoints.rs

```diff
@@ -7386,13 +7390,49 @@ async fn user_list(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+/// List users
+#[endpoint {
+    method = GET,
+    path = "/v1/users",
+    tags = ["silos"],
+}]
+async fn user_list_v1(
+    rqctx: RequestContext<Arc<ServerContext>>,
+    query_params: Query<PaginatedById<params::OptionalGroupSelector>>,
```

Comment:
Is the group selector actually optional?

Reply:
The way I pictured it working was: if you don't specify a group, you get all users. If you specify a group you get the users in that group.